docs/CHANGELOG_UI_CALLING.md

```diff
@@ -1,5 +1,11 @@
 # Azure Communication UI Calling Release History
 
+## 1.1.0 (2022-11-XX)
+- This version is the public GA release with Calling UI Library containing the features and bug fixes from 1.1.0-beta.1 refresh.
```

Comment:
I think we can leave this description out.

Since ChangeLog is read top->down, we don't need to really say "look at next item", it's where eyes will land.

I think developers generally understand that 1.1.0 includes 1.1.0-beta.xx, so if it's just a line item "released" people will get it.